### DISCUSSION TITLE
ubuntu 18.04 ardupilot kurdum ancak python sürümü hata veriyor

### DISCUSSION THREAD

**Discussion by jrnsng** [community/community#67403] [Programming Help] [answered]
### Select Topic Area

Question

### Body

../Tools/autotest/sim_vehicle.py -w --console --map
başlatmak için kullandığım kod bu ancak başlatırken şununla karşılaşıyorum:
**_SIM_VEHICLE: Start
SIM_VEHICLE: Killing tasks
SIM_VEHICLE: Starting up at SITL location
SIM_VEHICLE: WAF build
SIM_VEHICLE: Configure waf
SIM_VEHICLE: "/home/yusuf/ardupilot/modules/waf/waf-light" "configure" "--board" "sitl"
Setting top to                           : /home/yusuf/ardupilot 
Setting out to                           : /home/yusuf/ardupilot/build 
Autoconfiguration                        : enabled 
Checking for program 'python'            : /usr/bin/python 
Checking for python version >= 3.6.9     : 2.7.17 
The python version is too old, expecting (3, 6, 9)
(complete log in /home/yusuf/ardupilot/build/config.log)
SIM_VEHICLE: (Configure waf) exited with code 256
SIM_VEHICLE: Killing tasks_**
burda python sürümünün yeterlil olmadığını belirtiyo
ek olarak da python -V yazdığımda 2.7.17 vermesine rağmen 
python3 -V yazdığımda ise 3.6.9 yani gerekli sürüm gözüküyor buna rağmen çözemiyorum

**Comment by ilker0** [comment]
`python3 ../Tools/autotest/sim_vehicle.py -w --console --map` şeklinde denediniz mi ?

**Reply by jrnsng** [reply]
aynı hatayı aldım bir şey değişmedi
**_
SIM_VEHICLE: Start
SIM_VEHICLE: Killing tasks
SIM_VEHICLE: Starting up at SITL location
SIM_VEHICLE: WAF build
SIM_VEHICLE: Configure waf
SIM_VEHICLE: "/home/yusuf/ardupilot/modules/waf/waf-light" "configure" "--board" "sitl"
Setting top to : /home/yusuf/ardupilot
Setting out to : /home/yusuf/ardupilot/build
Autoconfiguration : enabled
Checking for program 'python' : /usr/bin/python
Checking for python version >= 3.6.9 : 2.7.17
The python version is too old, expecting (3, 6, 9)
(complete log in /home/yusuf/ardupilot/build/config.log)
SIM_VEHICLE: (Configure waf) exited with code 256
SIM_VEHICLE: Killing tasks_**

**Reply by nurankalan** [reply]
aynı hatayı ben de alıyorum çözümünü bulabildin mi acaba

**Reply by sami-cmd01** [reply]
Bulabildin mi hatayı

**Comment by jrnsng** [comment, marked as answer]
cozumunu bulamadim ama ubuntu yerine windows kullaniyorum cozuldu hata

21 Eki 2023 Cmt 15:02 tarihinde nurankalan ***@***.***> şunu
yazdı:

**Comment by BetulSare** [comment]
windows'ta nasıl yaptın anlatır mısın

**Comment by jrnsng** [comment]
dronekit yerine missionplanner kullaniyorum (ne yazdigimi hatirlamiyorum
umarim sordugun budur)
